javascript-data-model/order-history.js

```diff
@@ -0,0 +1,40 @@
+const orderHistory = [
+  {
+    product1: 'Javascript for impatient programmers',
+    author: 'Rauschmayer, Dr. Axel',
+    datePlaced: 'Aug 4, 2020',
+    dateDelivered: 'Aug 8, 2020',
+    total: 34.0,
+    shipTo: 'JS Master',
+    returnWindowClosed: 'Sept 7, 2020',
+  },
+  {
+    product1: 'The Timeless Way of Building',
+    author: 'Alexander, Christopher',
+    datePlaced: 'July 19,2020',
+    dateDelivered: 'July 20, 2020',
+    total: 44.53,
+    shipTo: 'JS Master',
+    returnWindowClosed: 'Aug 19, 2020',
+  },
+  {
+    product1:
+      'Gamecube controller Adapter. Super smash bros switch gamecube adapter for WII U, PC. Support turbo and vibration features. No driver and no lag gamecube adapter.',
+    datePlaced: 'July 4, 2020',
+    dateDelivered: 'July 7, 2020',
+    total: 17.22,
+    shipTo: 'JS Master',
+    returnWindowClosed: 'Aug 5, 2020',
+  },
+  {
+    product1: 'Gamecube controller- supersmash bros edition(Nintendo Switch',
+    price1: 94.95,
+    product2: 'The Art of Sql',
+    price2: 33.99,
```

Comment:
Rather than breaking these up into individual properties, this would be a great example of when an array would be really useful. Since we have a collection of items, but they are all apart of the same order, we can use an array of objects! Check out my snippet below.

```
items: [{
        type: 'hardware',
        name: 'GameCube Controller - Super Smash Bros. Edition (Nintendo Switch)',
        canReturnUntil: '08/04/2020',
        price: 94.95,
        imageUrl: 'images/gc-controller.png',
    },
    {
        type: 'book',
        title: 'The Art of SQL',
        author: 'Faroult, Stephane',
        canReturnUntil: '08/04/2020',
        price: 33.99,
        imageUrl: 'images/sql-book.png',
    },
]